test-integration/src/test/java/nl/nn/adapterframework/filesystem/ExchangeFileSystemTest.java

```diff
@@ -75,16 +76,19 @@ public void testMessageIdDoesNotChangeWhenMessageIsMoved() throws Exception {
 		Map<String,Object> threadContext1 = new HashMap<>();
 		Map<String,Object> threadContext2 = new HashMap<>();
 
-		ExchangeMessageReference msg1 = listener1.getRawMessage(threadContext1);
-		String msgId1 = listener1.getIdFromRawMessage(msg1, threadContext1);
+		RawMessageWrapper<ExchangeMessageReference> msg1 = listener1.getRawMessage(threadContext1);
+		Map<String,Object> messageContext1 = listener1.populateContextFromMessage(msg1.getRawMessage(), threadContext1);
+		String msgId1 = (String) messageContext1.get("mid");
```

Comment:
Heel eigenlijk zou je hier de PipelineSession.messageIdKey moeten gebruiken. Dit kan ook in een later PR, maar dan wel graag hier een issue voor aanmaken zodat we het niet vergeten 👍🏻

Reply:
Oh, dat moet inderdaad wel ja. Dit zijn die niet-werkende integration tests.